pkg/image/apis/image/v1/trigger/trigger.go

```diff
@@ -3,7 +3,7 @@ package trigger
 // TriggerAnnotationKey is the annotation used on resources to signal they wish to have
 // container image references changed when an image stream tag is updated. Today, only
 // containers can be specified by fieldPath.
-const TriggerAnnotationKey = "image.alpha.openshift.io/triggers"
+const TriggerAnnotationKey = "image.openshift.io/triggers"
```

Comment:
I'll point out here, what @mfojtik pointed in the comments. Are you thinking of existing users? Do you plan some kind of migration/release notes about this change. Why not supporting both of the annotation for a release for a smooth migration and change only after?

Reply:
My current thinking is how #16225 does it